source/blog/2016-09-05-last-week-in-manageiq-api-refactor-observables.html.md

```diff
@@ -0,0 +1,81 @@
+
+
+## Improved
+
+### [Convert API collections to individual controllers](https://github.com/ManageIQ/manageiq/pull/10941)
+
+[Tim Wade](https://github.com/imtayadeway) has done an impressive amount of work refactoring our REST API code, [converting collections to individual controllers](https://github.com/ManageIQ/manageiq/pull/10890), [moving the entrypoint to ApiController](https://github.com/ManageIQ/manageiq/pull/10880), [changing the namespace](https://github.com/ManageIQ/manageiq/pull/10823) and [removing custom routing](https://github.com/ManageIQ/manageiq/pull/10993), which may make it much easier to expand the API to manage *all* the things, REST style :).
+
+### [Add missing gettext](https://github.com/ManageIQ/manageiq/pull/10935)
+[Milan Zazrivec](https://github.com/mzazrivec) is pushing forward with our i18n efforts, this week by adding missing gettext, *and* [restoring internationalization functionality in toolbars](https://github.com/ManageIQ/manageiq/pull/10787). If you want him to keep his sanity, please go through our [i18n guidelines](https://github.com/ManageIQ/guides/blob/master/i18n.md) at least once a week, before bedtime.
+
+### [For Middleware Server operations use Rx.js Observables for communication](https://github.com/ManageIQ/manageiq/pull/10687)
+[Mike Thompson](https://github.com/mtho11) has been working in middleware code, this time to use [RxJs Observables](https://github.com/Reactive-Extensions/RxJS/blob/master/doc/api/core/observable.md) to talk to the toolbar, progress! :)
+
+
+## Fixed
+
+### [Fix dev reload issues when Vmdb::Settings.reload! is called](https://github.com/ManageIQ/manageiq/pull/10988)
+[Jason Frey](https://github.com/Fryguy) and [Chris Arcand](https://github.com/chrisarcand) have been busy in fixing-the-fix-to-fix-the-fix-land - this time to fix some reload issues and test failures after the recent changes to Vmdb::Settings. First, we [skipped the failures](https://github.com/ManageIQ/manageiq/pull/10841), [actually skipped the failures](https://github.com/ManageIQ/manageiq/pull/10857), then [fixed the failures](https://github.com/ManageIQ/manageiq/pull/10886) and the reload error in devel mode. Developers everywhere, rejoice!
+
+### [Check the request format before setting the redirection after the login page](https://github.com/ManageIQ/manageiq/pull/10702)
+[Yaacov Zamir](https://github.com/yaacov) has done a small but important fix to prevent an annoying routing error, when the user gets redirected to the login screen from a POST action. 1 error less, `NaN` to go!
+
+### [Force jasmine-core 2.4.1 instead of 2.5](https://github.com/ManageIQ/manageiq/pull/10898)
+Last week [jasmine 2.5](https://github.com/jasmine/jasmine) also came out, making all our javascript tests to fail because of an overly optimistic version constraint. Fixed shortly after by [yours truly](https://github.com/himdel), *cough* shameless plug *cough*.
```

Comment:
I think either "making all our [...] tests fail" or "causing all our [...] tests to fail" :smile: